biogeochem/FatesAllometryMod.F90

```diff
@@ -522,6 +534,12 @@ subroutine carea_allom(dbh,nplant,site_spread,ipft,crowndamage,c_area,inverse)
           call carea_2pwr(dbh_eff,site_spread,d2bl_p2,d2bl_ediff,d2ca_min,d2ca_max, &
                crowndamage, c_area, do_inverse)
           capped_allom = .true.
+       case (4)
+          dbh_eff = min(dbh,dbh_maxh)
```

Comment:
Noting that height capping is compatible with this scheme and implementation makes sense.